java/src/jmri/jmrix/can/cbus/node/CbusNode.java

```diff
@@ -1239,7 +1239,8 @@ protected void sendNextEvVarToFetch() {
                 int index = _nodeEvents.get(i).getIndex();
                 int nextevvar = _nodeEvents.get(i).getNextOutstanding();
                 
-                if ( index > 0 ) {
+                // index from NERD / ENRSP indexing may start at 0
+                if ( index > -1 ) {
```

Comment:
Just a personal thought and nothing important.

But I think it would be good to change it to  `index >= 0` instead of `index > -1`. It would make the code easier to read.

Reply:
Hi Dan,
Thanks for the input, I see what you mean.
In this case, the -1 is the important thing, ( -1 meaning there is definitely should be a value for it but it's not yet known ) not the 0.
With this in mind, I find it easier to scan relative to the -1 condition rather than a 0 , it's the start of an else statement, so the -1 is the key value being tested for boolean, if that makes sense?